### PR TITLE
License headers for .proto files

### PR DESCRIPTION
**Pull request by daviddrysdale**

### CODE REVIEW

File: scripts/presubmit.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ find . -name "mock_*" -delete
 golint --set_exit_status ./...
 go vet ./...
 misspell -error -locale US .
-nolicense=$(find . -name "*.go" | xargs grep -L "Apache License")
+nolicense=$(find . -path ./protoc -prune -name "*.go" -o -name "*.proto" | xargs grep -L "Apache License")
```
Comment (codingllama) [inline scripts/presubmit.sh]: If I run this locally I don't get any ".go" files in the output.

The best I got was `find . -name '*.go' -or -name '*.proto' | grep -v ./protoc`.

(Btw, nit: single quotes for '*.go' and '*.proto')

Comment (daviddrysdale) [inline scripts/presubmit.sh]: Serves me right for blindly applying something from Stack Overflow...

Comment (codingllama) [inline scripts/presubmit.sh]: OK, here's the find incantation that I think works:

`find . -path ./protoc -prune -o -name '*.proto' -or -name '*.go' -print`

Comment (daviddrysdale) [inline scripts/presubmit.sh]: I think maybe stick to your first suggestion, to avoid everyone having to look up obscure `find` options...

**Pull request merged**